python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -344,6 +345,7 @@ class CalibTask(BatchPoolTask):
     RunnerClass = CalibTaskRunner
     filterName = None
     calibName = None
+    exposureTime = 1.0                  # sets this exposureTime in the output
```

Comment:
You want to give flats a unit exposure time? It might be helpful to give them the mean exposure time of the inputs.